ownCloudAppShared/AppLock/AppLockManager.swift

```diff
@@ -373,6 +381,7 @@ public class AppLockManager: NSObject {
 				let delayUntilNextAttempt = pow(powBaseDelay, Double(failedPasscodeAttempts))
 
 				lockedUntilDate = Date().addingTimeInterval(delayUntilNextAttempt)
+				lockSetTime = ProcessInfo.processInfo.systemUptime
```

Comment:
`lockSetTime` is set if an unlock fails, but is not removed in case of a successful unlock.